### ISSUE TITLE
[2.3 Preview] Chat Feed uses wrong Discord channel input

### ISSUE THREAD

**Issue by drkshrk** [Eco-DiscordLink/EcoDiscordPlugin#52]
Chat integration on bleeding edge stopped posting from the right discord channel.  I have deleted the config and reset it up to make sure it wasnt a bugged config.

the in-game text is posting to the correct channel but the discord -> game is posting from the wrong discord channel.  it uses the channel names general in discord regardless of what i put in the config.

**Comment by MonzUn** [comment]
Thanks for the report!
I've done a lot of refactoring recently and EcoWorld still runs 2.2.1 to see that it's stable in Eco 9.3.4.

Does it always use the wrong channel, or did it switch during runtime or at a restart?

**Comment by drkshrk** [comment]
Yeah it always ignores the config and posts from discord general Chanel to ingame instead of how it's set in config.

The in-game to discord is posting correctly in the discord.

For reference here's the screen shot of the config![image](https://user-images.githubusercontent.com/2175749/117816496-90a97400-b234-11eb-903e-5fe74d507eb6.png)

My assumption is somewhere the reference got flipped and it's using the EcoChannel setting for the discord.  I haven't seen/understood the code on where it might be pulling wrong.  Before you changed the confits it worked fine but when I pulled the changes that changed confits and stuff I ran into this.. then again I haven't put any real time other than some quick pokes to see if something stood out
